### PR TITLE
Update update workflows

### LINKED ISSUE #327: support for java20
can you please tell the estimated time by which we will get the support for graalvm20

### PR DESCRIPTION
**Pull request by dmikusa**
## Summary

1. Remove Java 11. This has gone end of life upstream, so it is being removed.
2. Remove Java 19. This has gone end of life upstream, so it is being removed.
3. Add Java 20. This is the new short-term release.
4. Update Java 17 to use the new Github release update dependency. We can now use this to fetch dependencies from upstream instead of the custom updater. This works for Java 17 and Java 20.

## Use Cases

Fix updaters.

Resolves #327 

## Checklist

* [ ] I have viewed, signed, and submitted the Contributor License Agreement.
* [ ] I have linked issue(s) that this PR should close using keywords or the Github UI (See [docs](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue))
* [ ] I have added an integration test, if necessary.
* [ ] I have reviewed the [styleguide](https://github.com/paketo-buildpacks/community/blob/main/STYLEGUIDE.md) for guidance on my code quality.
* [ ] I'm happy with the commit history on this PR (I have rebased/squashed as needed).

### CODE REVIEW

**Review by anthonydahanne — APPROVED** [review]
This change introduces new archives for GraalVM 17 and 20, that now include `native-image`.
Before we can consider `native-image` as part of GraalVM, like @dmikusa said, 11 must be monitored or... deprecated... since it seems to not be distributed "officially" anymore, see: https://www.graalvm.org/downloads/

**Comment by dmikusa** [comment]
This needs a little more work. I missed required changes to `buildpack.toml`. Marking as a draft for now.

**Comment by dmikusa** [comment]
I think this is ready to go now. Need a quick re-review. Thanks

File: .github/pipeline-descriptor.yml [hunk 1]
```diff
@@ -20,60 +20,27 @@ docker_credentials:
...
-  with:
-    glob:    native-image-installable-svm-java11-linux-amd64-.+.jar
-    token:   ${{ secrets.PAKETO_BOT_GITHUB_TOKEN }}
-    version: 11
 - name:            JDK 17
   id:              jdk
   version_pattern: "17\\.[\\d]+\\.[\\d]+"
   purl_pattern:    "17\\.[\\d]+\\.[\\d]+"
-  uses:            docker://ghcr.io/paketo-buildpacks/actions/graalvm-dependency:main
+  uses:            docker://ghcr.io/paketo-buildpacks/actions/github-release-dependency:main
```
Comment (pivotal-david-osullivan) [inline .github/pipeline-descriptor.yml:27]: Is there a reason you switched to github-release action for 17 and not for 20?

Comment (dmikusa) [reply .github/pipeline-descriptor.yml:27]: Copy & paste error. :) Good catch. I'll fix that.

**Review by pivotal-david-osullivan — APPROVED** [review]

**Pull request merged**